release.py

```diff
@@ -310,17 +310,25 @@ def validate_release_num(version):
         fail("The specified version is not a valid release version number")
     validate_release_version_parts(version)
 
+def sanitize_input(input_msg: str) -> str:
+    """Sanitize inputs from users. This removes leading and trailing spaces.
+       Use this function instead of input where user input is needed.
+    """
+    input_from_user = input(input_msg)
+    return input_from_user.replace(' ', '')
```

Comment:
correct me if I am wrong, wouldn't it replace all spaces (instead of just the trailing and leading ones?). There are legitimate use cases to have spaces in the middle of words, e.g. when we ask for name in gpg signature input.

Reply:
Yes, my bad, I assumed it would always be a one line input, will modify this.